shiny/ui/datagrid/js/index.tsx

```diff
@@ -262,39 +262,11 @@ const roots = new WeakMap<HTMLElement, Root>();
 
 class ShinyDataGridBinding extends Shiny.OutputBinding {
   find(scope: HTMLElement | JQuery<HTMLElement>): JQuery<HTMLElement> {
-    return $(scope).find(".shiny-glide-data-grid-output");
+    return $(scope).find("shiny-glide-data-grid-output");
```

Comment:
Should there be a difference between `<shiny-glide-data-grid-output>` tags intended for static rendering, and intended for Shiny output binding?

Reply:
For Shiny outputs, there are three places we can take options:

1. Arguments to `output_foo()`, like `output_plot`'s `click` argument
2. Arguments to `render_foo()`, like `render.plot`'s `alt` argument
3. Attached to the object returned from the user's render code, like the `datatable()` in `DT` for R

In this case, I've been assuming it'd all be in 3 (although I'm not sure why). I think the answer does influence 👆

Reply:
> Should there be a difference between `<shiny-glide-data-grid-output>` tags intended for static rendering, and intended for Shiny output binding? (I'm asking)

I don't have strong feelings about it either way. It is nice to be able to reuse the `renderValue` code for both cases, but that could be done with class inheritance.

As for where to take options, that's a good question.

Maybe accept them in 1, and allow them to be overridden in 3? (We should be more consistent about this in general.)